bazel/ray_deps_setup.bzl

```diff
@@ -261,6 +261,7 @@ def ray_deps_setup():
         name = "rules_perl",
         remote = "https://github.com/bazelbuild/rules_perl.git",
         commit = "022b8daf2bb4836ac7a50e4a1d8ea056a3e1e403",
+        shallow_since = "1663780239 -0700",
```

Comment:
maybe put a link to bazel documentation?

Reply:
it's fairly standard thing for using `git_repository`: https://bazel.build/rules/lib/repo/git

the hint to add a `shallow_since` is provided by bazel build process in the build logs.

Reply:
added comments.